types/tags.go

```diff
@@ -51,3 +51,13 @@ func NewTags(tags ...interface{}) Tags {
 func MakeTag(k string, v []byte) Tag {
 	return Tag{Key: []byte(k), Value: v}
 }
+
+//__________________________________________________
+
+// common tags
+var (
```

Comment:
Note sure this file is the right place for these constants. Maybe in `x/staking`?

Reply:
it was intentional - I'd opt to keep them here until we create another thin later of abstraction (and move these tags there) for the group of modules surrounding the current DPOS implementation. For the current situation, we should enforce that governance/slashing/token-fountain/staking all use the same flag variables